plugin/README.md

```diff
@@ -0,0 +1,51 @@
+# lighthouse
+
+## Name
+
+*Lighthouse* - serve services across clusters.
```

Comment:
serve services? :-)
 
what about DNS discovery for services across clusters?

Reply:
Changed to discover, wording was from template README I picked up.